todo/views/auth.py

```
@@ -144,178 +137,77 @@ def _handle_callback_directly(self, code, request):
-        state = request.query_params.get("state")
-        error = request.query_params.get("error")
-
-        frontend_callback = f"{settings.FRONTEND_URL}/auth/callback"
-
-        if error:
-            return HttpResponseRedirect(f"{frontend_callback}?error={error}")
-        elif code and state:
-            return HttpResponseRedirect(f"{frontend_callback}?code={code}&state={state}")
-        else:
-            return HttpResponseRedirect(f"{frontend_callback}?error=missing_parameters")
+        except Exception:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)